TeamCode/src/main/java/org/firstinspires/ftc/teamcode/modules/robot/DriveToAprilTag.java

```diff
@@ -261,7 +204,7 @@ private void initAprilTag() {
      Manually set the camera gain and exposure.
      This can only be called AFTER calling initAprilTag(), and only works for Webcams;
     */
-    private void    setManualExposure(int exposureMS, int gain) {
+    private void    setManualExposure(int exposureMS, int gain, LinearOpMode op) {
```

Comment:
Passing in opMode does not work like you think it does, lets discuss.  There are also alternate / better ways to do this.  Additionally, lets talk through this class in general.